src/player/PlayerComponent.cpp

```diff
@@ -648,6 +655,49 @@ void PlayerComponent::handleMpvEvent(mpv_event *event)
       }
       break;
     }
+    case MPV_EVENT_HOOK:
+      // FIXME: do these actually need to be filtered on old libmpv versions?
+      if (mpv_client_api_version() >= MPV_MAKE_VERSION(1, 108))
+      {
+	mpv_event_hook *hook = (mpv_event_hook *)event->data;
```

Comment:
Since you're declaring a local here, you'll still need the block (just with no need for the runtime `if`); it should be indented at the same level as the `case`, as with the other cases.